typescript/infra/scripts/helloworld/kathy.ts

```diff
@@ -304,8 +381,7 @@ async function updateWalletBalanceMetricFor(
 
 main()
   .then(() => {
-    error('Main exited');
-    process.exit(1);
```

Comment:
Uh, we need this still. There's a non-zero chance that even in the case that it is only running once that it will have some outstanding promise somewhere that keeps things from closing down. We at least need a `process.exit(0)`. One step better is exit 1 if it was not supposed to run only once.

Reply:
ok, gonna `process.exit(0);` just because plumbing the `cycleOnce` into this level would be awkward. It's also clear if kathy stops running while it's in statefulset mode